apps/web/e2e/helpers/debug-scripts/fix-electron-paths-simple.js

```
@@ -0,0 +1,415 @@
+  return findFilesByType(dir, ['.html']);
+}
+
+// =================== VALIDATION FUNCTION - PHASE 3 ===================
+// Comprehensive path validation
+function validatePaths(content, filename) {
+  const issues = [];
+  
+  // Check for remaining absolute paths
+  const absolutePathRegex = /(?:href|src|url)=["']\/((?!http)[^"']+)["']/g;
+  let match;
+  while ((match = absolutePathRegex.exec(content)) !== null) {
```

Biome (2.1.2)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)